features/channel/managing_channels/denying_access_to_channels_for_unauthorized_users.feature

```diff
@@ -0,0 +1,21 @@
+@managing_channels
+Feature: Denying access to shipping methods for unauthorized users
+    In order to denies access for unauthorized users
+    As a Visitor
+    I don't want to access to manage shipping methods
```

Comment:
I don't think a visitor should be a subject here. 

```suggestion
    In order to secure the channels management
    As an Administrator
    I want to prevent accessing the channel management by unauthorized users
```

Reply:
`As visitor` probably logs-in with shop firewall and hence the error on modification, admin user should be able to remove it.

Reply:
I understand, but as a user, it's not my business someone didn't secure their shop. The "Stakeholder" of this feature is a Shop Owner / Administrator / whatever you name it, as it's their problem if someone unauthorized modifies the channel :D.